event-grid/subscribe-to-azure-subscription-preview/subscribe-to-azure-subscription-preview.ps1

```diff
@@ -0,0 +1,15 @@
+# You must have the latest version of the Event Grid PowerShell module.
+# To install:
+# Install-Module -Name AzureRM.EventGrid -AllowPrerelease -Force -Repository PSGallery
+
+# Provide an endpoint for handling the events.
+$myEndpoint = "<endpoint URL>"
+
+# Select the Azure subscription you want to subscribe to.
+Set-AzureRmContext -Subscription "Contoso Subscription"
+
+# Get the subscription ID
+$subID = (Get-AzureRmSubscription -SubscriptionName "Contoso Subscription").Id
```

Comment:
This is a repeat of the previous command. Instead assign the value from the initial call to `$subID`.